cirq/google/gate_sets.py

```diff
@@ -67,112 +71,14 @@ def _near_mod_2pi(e, t, atol=1e-8):
-                        gate_getter=protocols.measurement_key),
-                    op_serializer.SerializingArg(serialized_name='invert_mask',
-                                                 serialized_type=List[bool],
-                                                 gate_getter='invert_mask')
-                ])
-        ],
-        deserializers=[
-            op_deserializer.GateOpDeserializer(
-                serialized_gate_id='exp_w',
-                gate_constructor=ops.PhasedXPowGate,
-                args=[
-                    op_deserializer.DeserializingArg(
-                        serialized_name='axis_half_turns',
-                        constructor_arg_name='phase_exponent'),
-                    op_deserializer.DeserializingArg(
-                        serialized_name='half_turns',
-                        constructor_arg_name='exponent')
-                ]),
-            op_deserializer.GateOpDeserializer(
-                serialized_gate_id='exp_z',
-                gate_constructor=ops.ZPowGate,
-                args=[
-                    op_deserializer.DeserializingArg(
-                        serialized_name='half_turns',
-                        constructor_arg_name='exponent')
-                ]),
-            op_deserializer.GateOpDeserializer(
-                serialized_gate_id='exp_11',
-                gate_constructor=ops.CZPowGate,
-                args=[
-                    op_deserializer.DeserializingArg(
-                        serialized_name='half_turns',
-                        constructor_arg_name='exponent')
-                ]),
-            op_deserializer.GateOpDeserializer(
-                serialized_gate_id='meas',
-                gate_constructor=ops.MeasurementGate,
-                args=[
-                    op_deserializer.DeserializingArg(
-                        serialized_name='key', constructor_arg_name='key'),
-                    op_deserializer.DeserializingArg(
-                        serialized_name='invert_mask',
-                        constructor_arg_name='invert_mask',
-                        value_func=lambda x: tuple(cast(list, x)))
-                ],
-                num_qubits_param='num_qubits'),
-        ]))
+    serializable_gate_set.SerializableGateSet(gate_set_name='xmon',
+                                              serializers=[
+                                                  *SINGLE_QUBIT_SERIALIZERS,
+                                                  CZ_SERIALIZER,
+                                                  MEASUREMENT_SERIALIZER,
+                                              ],
+                                              deserializers=[
+                                                  *SINGLE_QUBIT_DESERIALIZERS,
+                                                  CZ_DESERIALIZER,
+                                                  MEASUREMENT_DESERIALIZER,
+                                              ]))
```

Comment:
nit: add trailing comma after the last argument to make this format better. Also, is the type annotation needed? If not, can put the constructor on the first line and remove the outer parens.

Reply:
Done.  

Without the type annotation, I get a bunch of mypy warnings that I couldn't figure out how to get rid of otherwise.

```
cirq/google/engine/engine.py:227: error: Cannot determine type of 'XMON'
cirq/google/engine/engine.py:284: error: Cannot determine type of 'XMON'
cirq/google/engine/engine.py:328: error: Cannot determine type of 'XMON'
cirq/google/engine/engine.py:466: error: Cannot determine type of 'XMON'
cirq/google/engine/engine.py:499: error: Cannot determine type of 'XMON'
```

Reply:
gate_set should be an attribute on the `Circuit` imo